lib/tasks/taxonomy/analysis/taxons_tagged_to_content.rake

```diff
@@ -24,22 +24,33 @@ namespace :taxonomy do
         page += 1
       end
 
-      puts %w(organisations content_id base_path number_of_taxons).to_csv
+      content_items.each do |item|
```

Comment:
I think this loop can be happily merged into the loop below, which also iterates over `content_items`? Then you don't have to actually mutate the `content_item` (which can be a little surprising).

You'd also then be able to directly use the `publishing_api` response when constructing your `csv_row`. For example:

```ruby
# Set up csv_row (omitted)
csv_row = [...]

# Fetch taxon data
taxons = item['taxons'].map do |taxon_id|
  Services.publishing_api.get_content(taxon_id)
end

# Reformat data for extra columns
taxon_columns = taxons.each_with_object([]) do |taxon, columns|
  columns << taxon['content_id']
  columns << taxon['base_path']
end

# Add extra columns
csv_row += taxon_columns
```